src/editEngine.js

```diff
@@ -72,15 +100,47 @@ export const addRow = ({ editInfo = generateInitialEditInfo(), editedRow }) =>
   updateRow({ editInfo, editedRow })
 
 export const batchUpdateRow = ({ editInfo = generateInitialEditInfo(), updates }) => {
-  const { history } = editInfo
-  let finalEdit = editInfo
+  const { added, dirtyMap, updatedMap, updated, history, ...rest } = editInfo
```

Comment:
was this not working? or this is for performance reasons?  And i would prefer to have this block of code tested

Reply:
yeah, i changed it for performance reason, now it can handle tens of thousands of rows together